backend/services/locationService.test.ts

```diff
@@ -105,4 +105,11 @@ describe('Location service', () => {
       expect(manageCourtsService.getCourt).toBeCalledWith(context, 'LDNCOU')
     })
   })
```

Comment:
The request booking journey is slightly different to the create, update and delete appointment journeys in that the call to the notification service to send emails is done directly in the offenderDetailsController rather than in a 'service' .
The create, update and delete journeys get the court email address by calling whereabouts in their respective services. As the request journey doesn't have a dedicated service, I have created a function within the locationService to get the court email address. Could arguably have created a new service for the request journey even though it would have only a single function in it.

Reply:
following discussion with Andy, have now moved the call to get email address and send email, into a new requestService.